bitbots_ros_control/scripts/foot_pressure_tester.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+import rospy
+from bitbots_msgs.msg import FootPressure
+import argparse
+import random
+import time
+import tkinter
+import threading
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("-r", "--rate", help="Publish rate", dest="rate", type=int, default=200)
+parser.add_argument("-n", "--noise", help="Amount of noise on the signal", type=float, default=1e6)
+parser.add_argument("-s", "--seed", help="random seed for zero and scale", dest="seed")
+
+args = parser.parse_args()
+
+if args.seed is not None:
+    random.seed(args.seed)
+
+zeroes = [2e8 * random.random() - 1e8 for i in range(8)]
+scales = []
+for i in range(8):
+    if random.random() < 0.5:
+        m = 1
+    else:
+        m = -1
+    scales.append((random.random() * 2e7 + 1e7) * m)  # random value between 1e7 and 3e7 or -1e7 and -3e7
+
+force_values = [0] * 8
+
+
+# this is ugly but I dont have patience to make it properly
+
+
+def update_force_0(force):
+    force_values[0] = float(force)
+
+
+def update_force_1(force):
+    force_values[1] = float(force)
+
+
+def update_force_2(force):
+    force_values[2] = float(force)
+
+
+def update_force_3(force):
+    force_values[3] = float(force)
+
+
+def update_force_4(force):
+    force_values[4] = float(force)
+
+
+def update_force_5(force):
+    force_values[5] = float(force)
+
+
+def update_force_6(force):
+    force_values[6] = float(force)
+
+
+def update_force_7(force):
+    force_values[7] = float(force)
+
```

Comment:
I would like to solve this more elegantly but I could not find out how to do this in python.

Reply:
cant you just add a paramter for the number and then call it with the "i"

Reply:
the problem is that I need to pass the function to the slider widget